README.md

```diff
@@ -702,6 +702,14 @@ const { results } = await Calculator.calculateMeasureReports(measureBundle, pati
 
 **NOTE**: Measure logic highlighting is not available when calculating a population `MeasureReport` with `reportType = 'summary'`
 
+However, highlighted HTML provided by `fqm-execution` is not solely based on whether individual pieces of the measure logic CQL had "truthy" or "falsy" values. When there are populations that depend on each other, a relevance hierarchy is built to ensure that populations that are subsets of others aren't considered for membership until membership is determined for their superset population. If a statement's superset statement is false, then no highlighting will appear, regardless of whether that statement's value is "truthy" or "falsy". This pattern reflects the actual population results that are being returned during calculation.
```

Comment:
Might want to consider separating this part out by a sub-heading or something, it's a lot of text to appear just under the previous explanation of measure highlighting. Maybe `### Population Relevance for Highlighting` or something. You don't have to include that section in the table of contents or anything, but just to break up the content a bit

Reply:
Good call, I always have trouble coming up with good sub-heading names so thank you!